lib/RoleRevokerResolver.ts

```diff
@@ -58,11 +58,10 @@ export class RoleRevokerResolver implements RevokerResolver {
       Logger.info(
         `IRevokerDefinition for namespace: ${namespace} fetched from SSI-Hub`
       );
-      return (
-        await this._cacheServerClient.getRoleDefinition({
-          namespace: resolvedNamespace,
-        })
-      ).revoker;
+      const roleDef = await this._cacheServerClient.getRoleDefinition({
+        namespace: resolvedNamespace,
+      });
+      return roleDef.issuer;
```

Comment:
@nichonien should it be `revoker` here?
```suggestion
      return roleDef.revoker;
```